src/fitnesse/slim/StatementExecutor.java

```diff
@@ -131,7 +140,52 @@ private MethodExecutionResult getMethodExecutionResult(String instanceName, Stri
     }
     return results.getFirstResult();
   }
+ 
+  /**
+   *  
+   * @return true is the fixture will handles symbols assignments and lookups itself.
+   *         This should be a rare exception and is not recommended
+   *         
+   *  Would be nice to use the classname but we don't know it at this point.
+   *         
+   */
+  
+  private Boolean ignoreSymbols(String instanceName, String methodName){
+	  try{
```

Comment:
This method (and the next) catch generic Exceptions. Though since the null values are handled, it's not clear to me what else could throw an exception here. Could you elaborate, and perhaps catch a more specific Exception? :)